.env

```diff
@@ -0,0 +1 @@
+PUBLIC_IP=
```

Comment:
Is the .env filename special in any way? It feels odd to have this as a hidden file.